app/views/docs/keys.phtml

```diff
@@ -4,9 +4,12 @@ $scopes = $this->getParam('scopes', );
 
 <p>Using your API Keys, you can access Appwrite services using the SDK of your choice. To create a new API key, go to your API keys tab in your project setting using your Appwrite console and click the 'Add API Key' button.</p>
 
-<p>When adding a new API Key, you can choose which permission scope to grant your application. It is a best practice to allow only the permissions you need to meet your project goals.</p>
+<p>When adding a new API Key, you can choose which permission scope to grant your application. If you need to replace your API Key, create a new key, update your app credentials and, once ready, delete your old key.</p>
```

Comment:
What do you think about just 'scopes' instead of 'permission scope'? It could be confusing with the permissions system being unrelated.

Or maybe 'resource scopes' instead?

cc @eldadfux

Reply:
Let's just use scopes :) I think it's standard to just say scopes in permission. It's confusing but a norm.